nanoFramework.Json/JsonObject.cs

```diff
@@ -246,7 +246,7 @@ public static JsonArray Serialize(ArrayList source)
 						result[index] = Serialize(valueType, item);
 					}
 				}
-
+				
```

Comment:
Not on the test file. Here a line that should be just a line.

Reply:
almost all files are tabs! I can fix that in another PR